### ISSUE TITLE
Some images get lost after reading and writing xlsx-template file

### ISSUE THREAD

**Issue by basementmedia2** [PHPOffice/PhpSpreadsheet#676]
This is:

```
- [x ] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
When opening a template (which contains images), adding some text in an empty cell and saving the file, the images which are already contained in the template should not get deleted.

### What is the current behavior?
When opening a template (which contains images), adding some text in an empty cell and saving the file, SOME images (not all) and with-excel-created lines/Graphics (e.g. arrows) which are already contained in the template get deleted.

### What are the steps to reproduce?

Please provide a [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```php
   require 'vendor/autoload.php';
   use PhpOffice\PhpSpreadsheet\IOFactory;
   use PhpOffice\PhpSpreadsheet\Spreadsheet;
   use PhpOffice\PhpSpreadsheet\Writer\Xlsx;

   $reader = IOFactory::createReader('Xlsx');

   $inputFileName="mypath/myfile.xlsx";
   $outputFileName=$inputFileName;
   
   //Reading the XLS-File, which contains some images
   $spreadsheet = $reader->load($inputFileName);
   $sheet = $spreadsheet->getActiveSheet();

   // Write Text in Cell
   $sheet->setCellValue('B28', "Thats a test");

  // Overwrite the file (which is made here pseudo by deleting the original file and writing the new file with same name)
  // Save new file with new name
  $writer = new Xlsx($spreadsheet);
  $writer->save($outputFileName."_");
   // 2. Delete original file
  unlink($outputFileName);
  // Rename new file to original-filename
  rename($outputFileName."_", $outputFileName);
```

### Which versions of PhpSpreadsheet and PHP are affected?
PHP-Version: 7.2
PHPSpreadsheet-Version: April 2018 (where can i get the version number?)

**Comment by MAKS-dev** [comment]
Looks like there is a relation with issue 543 and/or 544... no solution there yet.

**Comment by shahimclt** [comment]
Might also be related to #661.

**Comment by oleibman** [comment]
Resolved by PR #1462 in May 2020. Leaving issue closed, removing stale label.
